kotskinds/Makefile

```diff
@@ -6,7 +6,7 @@ generate: controller-gen client-gen
 		object:headerFile=./hack/boilerplate.go.txt \
 		paths=./apis/...
 	$(CONTROLLER_GEN) \
-		crd \
+		crd:allowDangerousTypes=true \
```

Comment:
why is this needed?

Reply:
removed now, but was added when the MappedChartValue struct's fields were exported (which was a bad idea)